src/main/psxsdk/libcd/c_011.c

```diff
@@ -2,6 +2,19 @@
 
 INCLUDE_ASM("main/nonmatchings/psxsdk/libcd/c_011", StCdInterrupt);
 
-INCLUDE_ASM("main/nonmatchings/psxsdk/libcd/c_011", mem2mem);
+void mem2mem(s32* arg0, s32* arg1, u32 arg2) {
+    s32 temp_v0;
+    s32* var_a0;
+    s32* var_a1;
+    u32 var_v1;
+
+    var_a0 = arg0;
+    var_a1 = arg1;
+
+    for (var_v1 = 0; var_v1 < arg2; var_v1++) {
+        temp_v0 = *var_a1++;
+        *var_a0++ = temp_v0;
+    }
+}
```

Comment:
```suggestion
void mem2mem(s32* dst, s32* src, u32 num) {
    u32 i;

    for(i = 0; i < num; i++) {
        *dst++ = *src++;        
    }
}
```
https://decomp.me/scratch/oxG4d